posts/physical-backup-restore.md

```diff
@@ -0,0 +1,457 @@
+---
+title: "Physical Backup & Restoration"
+description: "Physical Backup & Restoration"
+date: "2025-04-09"
+is_redirect: false
+redirect_url:
+---
+
+
+In Frappe Cloud, every **Site Update** triggers an automatic backup - ensuring your data stays safe in case of failures and we can rollback your site to older version as soon as possible.
```

Comment:
```suggestion
In Frappe Cloud, every site update triggers an automatic backup - ensuring your data stays safe in case of failures and we can rollback your site to older version as soon as possible.
```

I'd suggest toning down text formatting everywhere. It should be done sparingly.